### PR TITLE
added floors

### PR DESCRIPTION
**Pull request by kseniia-s**

### CODE REVIEW

File: src/Area.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+  public AreaType getType() {
+    return type;
+  }
+
+  public double getSquare() {
+    return square;
+  }
+
+  //  public static final String BEDROOM = "badroom";
```
Comment (VinShurikk) [inline src/Area.java]: Remove commented strings

File: src/Floor.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+import java.util.List;
+
+public class Floor {
+  private int floorNumber;
+  private int bedroomCount;
+  private double livingArea;
+  private double nonLivingArea;
+  private double totalArea;
+
+  List<Area> areas = new ArrayList<>();
```
Comment (VinShurikk) [inline src/Floor.java]: private field

File: src/House.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class House {
+  List<Floor> floors = new ArrayList<>();
```
Comment (VinShurikk) [inline src/House.java]: private field

File: src/House.java [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    floors.add(floor);
+    return floor;
+  }
+
+
+  public int getBedroomCount() {
+    //вычислить на каждом этаже сколько ванных комнат
+    for (int i = 0; i < floors.size(); i++){
+      Floor floor = floors.get(i);
+      List<Area> floorAreas = floor.areas;
```
Comment (VinShurikk) [inline src/House.java]: reuse already computed value.
floor.getBedroomCount();

File: src/House.java [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+    return bedroomCount;
+  }
+
+
+  public double getLivingArea() {
+  // посчитать на каждом этаже площать помещений типа ЛивингАреа
+    double houseLivingArea = 0.0;
+    for (int i = 0; i < floors.size(); i++){
+      Floor floor = floors.get(i);
+      List<Area> floorAreas = floor.areas;
```
Comment (VinShurikk) [inline src/House.java]: reuse already computed value.
floor.getLivingArea();

File: src/House.java [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+    }
+    return houseLivingArea;
+  }
+
+  public double getNonLivingArea() {
+    // посчитать на каждом этаже площать помещений типа NoЛивингАреа
+    double houseNonLivingArea = 0.0;
+    for (int i = 0; i < floors.size(); i++){
+      Floor floor = floors.get(i);
+      List<Area> floorAreas = floor.areas;
```
Comment (VinShurikk) [inline src/House.java]: reuse already computed value.
floor.getNonLivingArea();

File: src/Floor.java (hunk 2 above)
Comment (kseniia-s) [reply src/Floor.java]: what private field?

File: src/House.java (hunk 3 above)
Comment (kseniia-s) [reply src/House.java]: what private field?

File: src/Area.java (hunk 1 above)
Comment (kseniia-s) [reply src/Area.java]: removed

File: src/House.java (hunk 4 above)
Comment (kseniia-s) [reply src/House.java]: corrected

File: src/House.java (hunk 5 above)
Comment (kseniia-s) [reply src/House.java]: fixed

File: src/House.java (hunk 6 above)
Comment (kseniia-s) [reply src/House.java]: fixed

File: src/House.java (hunk 3 above)
Comment (kseniia-s) [reply src/House.java]: I got it. Fixed

File: src/Floor.java (hunk 2 above)
Comment (kseniia-s) [reply src/Floor.java]: I got it. Fixed

File: src/House.java (hunk 3 above)
Comment (VinShurikk) [reply src/House.java]: floors should be private

**Pull request merged**